tests/ssl_test.py

```diff
@@ -1,6 +1,4 @@
-import contextlib
 import random
-import socket
```

Comment:
As `eventlet.green.socket` is imported below I think it is safe to remove this one.